doc/chapter00.md

```diff
@@ -0,0 +1,244 @@
+
+<a name="1.1"></a>
+## Glossary
+The definition and intent of the terminology used throughout the documents is defined in the [Glossary](tech/glossary.md).
+
+<a name="1.2"></a>
+## Problem Statement
+Based on informal conversations with many operators and developers, there is a realisation that there are significant technical, operational and business challenges to the development and deployment of VNF/CNF applications related to the lack of a common cloud infrastructure platform.  These include but are not limited to the following:
+
+ - Higher development costs due to the need to develop Virtual Network Functions (VNF) on multiple custom platforms for each operator
+ - Increased complexities due to the need to maintain multiple versions of applications to support each custom environment
+ - Lack of testing and validation commonalities, leading to inefficiencies and increased time to market. While the operators will still do internal testing, but using an industry driven verification program based on a common cloud infrastructure would provide a head start.
+ - Slower adoption of cloud-native applications and architectures.  A Common Telco Cloud may provide an easier path to methodologies that will drive faster cloud-native development.
+ - Increased operational overhead due to the need for operators to integrate diverse and sometime conflicting cloud platform requirements.
+
+One of major challenges holding back the more rapid and widespread adoption of VNF is that the traditional telecom ecosystem vendors, while building or designing their virtualised services (whether it be Voice over LTE (VoLTE), Evolved Packet Core (EPC), or popular customer facing enterprise services such as SD-WAN (Software Defined Wide Area Network), are making their own infrastructure assumptions and requirements, often with custom design parameters. This leaves the operators being forced to build complex integrations of various vendor/function specific silos which are incompatible with each other and might possibly have different and conflicting operating models. In addition, this makes the onboarding and conformance processes of VNFs/CNFs (coming from different vendors) hard to automate and standardise.  
```

Comment:
since a phrase has been moved to be within parentheses, it needs to be reworded
```suggestion
One of the major challenges holding back the more rapid and widespread adoption of VNFs/CNFs is that the traditional telecom ecosystem vendors, while building or designing their virtualised services (for example, Voice over LTE (VoLTE), Evolved Packet Core (EPC), and SD-WAN (Software Defined Wide Area Network), are making their own infrastructure assumptions and requirements, often with custom design parameters. This leaves the operators being forced to build complex integrations of various vendor/function specific silos which are incompatible with each other and might possibly have different and conflicting operating models. In addition, this makes the onboarding and conformance processes of VNFs/CNFs (coming from different vendors) hard to automate and standardise.  
```